src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int rand = 101;
```

Comment:
Bad naming - rand. For example, MAX_BALL_NUMBER
Also, read constant convention: Constant names are to be written in uppercase with an underscore separating the words.